### PR TITLE
paris-project pizza done

### PR DESCRIPTION
**Pull request by filippo-fresilli**

### CODE REVIEW

**Comment by filippo-fresilli** [comment]
in collaboration with @enima2684

File: starter-code/index.html [hunk 1]
```diff
@@ -40,12 +40,12 @@ <h1>Pizza Builder</h1>
...
 
       <b>$10 cheese pizza</b>
-      <ul>
-        <li>$1 pepperonni</li>
-        <li>$1 mushrooms</li>
-        <li>$1 green peppers</li>
-        <li>$3 white sauce</li>
-        <li>$5 gluten-free crust</li>
+      <ul class="ingredient-list">
+        <li class="pep-price">$<span>1</span> pepperonni</li>
```
Comment (ta-web-paris) [inline starter-code/index.html:44]: I like how you added a span to target just the number

File: starter-code/pizza.js [hunk 2]
```diff
@@ -1 +1,69 @@
...
+    $(".crust").toggleClass("crust-gluten-free");
+    $(".btn-crust").toggleClass("active");
+    total()
+}); 
+
+
+function total() {
+    var totalPrice = 10;
+    
+ $(".ingredient-list li").each( function(index, li) {
```
Comment (ta-web-paris) [inline starter-code/pizza.js:57]: Good use of a function and the ```<span> ``` tag!

**Comment by ta-web-paris** [comment]
It looks good! I like how you created a total price function that you called on every click to update the DOM. Good use of the jQuery .toggle() 👏

**Pull request closed without merging**